source/retryable-writes/retryable-writes.rst

```diff
@@ -253,29 +248,23 @@ server may not support retryable writes if the
 cluster; however, that can only be reported as a server-side error (discussed
 later).
 
-Write commands specifying an unacknowledged write concern (i.e. ``{w: 0})`` are
-not supported. Drivers MUST raise an error if an unacknowledged write concern
-would be applied to any write command executed within a DriverSession where
-retryable writes have been enabled.
- 
-Drivers MAY raise the error for an unacknowledged write concern eagerly instead
-of waiting until a write operation is invoked. For example, drivers with an
-immutable collection object, which also do not allow a write concern to be
-specified on a per-operation basis, may prefer to raise an error at the time
-the collection is instantiated with an unacknowledged write concern from a
-DriverSession where retryable writes have been enabled.
-
-Unsupported Write Operations (server-side error)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Write commands specifying an unacknowledged write concern (e.g. ``{w: 0})``) do
+not support retryable behavior. Drivers MUST NOT add a transaction ID to any
+write command with an unacknowledged write concern executed within a MongoClient
+where retryable writes have been enabled. Drivers MUST NOT retry these commands
+if they fail to return a response.
```

Comment:
Delete the last sentence: these are unacknowledged.

Reply:
https://github.com/mongodb/specifications/pull/174/commits/a8feb4929050b860ea21e1ba6533d6e97d39b1e1 removes "if they fail to return a response".

Depending on how drivers execute unacknowledged ops, they may or may not return a response (smallest response likely being `{ok:1}`). It's more flexible to just not refer to a possible response.